frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -416,7 +416,7 @@ interactivetour-InteractiveTour--end-footer =
 ## Machinery
 ## Shows a list of translations from machines.
 machinery-Machinery--search-placeholder =
-    .placeholder = Search in Machinery
+    .placeholder = Search Translation Memory
```

Comment:
This will have to be "Concordance search", sorry spec. We also search other machinery sources, not just TM.